src/test/ml-modules/root/test/suites/merging/test-data/custom-merge-sjs.sjs

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+function maxMerge(propertyName, properties, propertySpec) {
```

Comment:
I know it's only a unit test, but that makes it an example. Since "maxMerge" checks whether to `goHigh` or not, maybe this should be "orderedMerge" instead.